components/jeune/DetailsJeune.tsx

```diff
@@ -57,6 +112,39 @@ export const DetailsJeune = ({
               <dd className='text-primary'>{jeune.email}</dd>
             </div>
           )}
+
+          {structureConseiller !== StructureConseiller.MILO && (
+            <div className='flex'>
+              <dt className='text-base-regular'>Identifiant Pôle Emploi :</dt>
+              <dd
+                className='text-base-bold ml-1'
+                onCopy={trackEventOnCopieIdentifiantPartenaire}
+              >
+                {identifiantPartenaire ? identifiantPartenaire : '-'}
+              </dd>
+              <button
+                className='ml-5 flex items-center text-primary'
+                aria-label={
+                  identifiantPartenaire
+                    ? 'Modifier l’identifiant pôle emploi'
```

Comment:
nitpick:
```suggestion
                    ? 'Modifier l’identifiant Pôle emploi'
```
https://pass-emploi.zeroheight.com/styleguide/s/75590/p/36c04e-lexique/b/7914a6

Reply:
Done